markdown/cmark/config.h.in

```diff
@@ -0,0 +1,76 @@
+#ifndef CMARK_CONFIG_H
+#define CMARK_CONFIG_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#cmakedefine HAVE_STDBOOL_H
+
+#ifdef HAVE_STDBOOL_H
+  #include <stdbool.h>
+#elif !defined(__cplusplus)
+  typedef char bool;
+#endif
```

Comment:
Trickier, because Autotools doesn't really have a clean way of force-including stuff, and this file is a mere *config.h*… one way could be having a second *config.h* just for cmark, but I'm not sure how to manage this with stock Autotools, short of manually generating that one (e.g. having a template and performing the replacements or using `AC_SUBST`/`AC_CONFIG_FILES`).